package.json

```diff
@@ -44,29 +46,49 @@
     "release:minor": "node scripts/release/release.js minor",
     "release:patch": "node scripts/release/release.js patch",
     "test": "npm run jest && npm run compile && npm run nightwatch && npm run nightwatch:port-provided && wdio wdio.conf.js",
+    "pack": "./bin/webpack.js --config src/webpack/webpack.config.js",
```

Comment:
this can be simplifed to `"pack": "webpack --config src/webpack/webpack.config.js",`